### DISCUSSION TITLE
MSP app intune deployment

### DISCUSSION THREAD

**Discussion by audian-benp** [KelvinTegelaar/CIPP#1343] [Ideas]
We have recently migrated to NinjaRMM and being able to deploy the NinjaRMM through inTune application deployment would make a great addition.

**Comment by dbaty7** [comment]
Agreed, or the ability to add our own app's for deployment within the MSP App section.

**Comment by BDOMichaelKirstNeshva** [comment]
Hi, I would like to deploy our own RMM App (customized).
How can I upload / integrate it?
Thanks
